### ISSUE TITLE
prompt does not work

### ISSUE THREAD

**Issue by essesesses** [enjalot/blockbuilder#180]
It appears that prompt does not work since last update
For demo, see
http://blockbuilder.org/anonymous/506ebf0e5377f920cbfe54c574e23025

**Comment by georules** [comment]
Did this work before?  I think it's a fair thing to block for security.  Although, allow-modals could be set to the sandbox to let this pass through.

**Comment by essesesses** [comment]
Yes it did. At least 6 days ago I was able to get prompt popups working.

I am not sure I can allow-modals directly from the block ?

**Comment by Fil** [comment]
I can confirm that adding `allow-modals ` works, also if that is the choice (and I support it), it would be necessary to remove `alertOverride` in `public/js/utils/parseCode.js`

See also #185.
